builtin/logical/consul/path_token.go

```diff
@@ -132,3 +138,39 @@ func (b *backend) pathTokenRead(ctx context.Context, req *logical.Request, d *fr
 
 	return s, nil
 }
+
+func parseServiceIdentities(data string) []*api.ACLServiceIdentity {
```

Comment:
It'd be nice to get some tests on these parsing functions to ensure they work as expected given various forms of input strings.

Reply:
Do you have any suggestions for what kind of approach you're thinking of for this?

Reply:
Discussed outside of this PR -- was thinking something like a table style unit test.